openpype/hosts/maya/plugins/publish/validate_rig_controllers.py

```diff
@@ -76,7 +76,20 @@ def get_invalid(cls, instance):
                 "All controls must be inside the rig's group."
             )
             return [controls_set]
+        skeleton_set = instance.data["rig_sets"].get("skeletonAnim_SET")
+        if not skeleton_set:
+            cls.log.info(
+                "No 'skeletonAnim_SET' in rig instance"
+            )
+        skeleton_controls = cmds.sets(skeleton_set, query=True)
+        if not all(control in lookup for control in cmds.ls(skeleton_controls,
+                                                            long=True)):
+            cls.log.error(
+                "All controls must be inside the rig's group."
+            )
+            return [skeleton_controls]
```

Comment:
Should we move this into a separate validator `validate_rig_skeleton_anim` since it's basically isolated from the regular controls for a rig, right?

Also, if the `skeleton_set` does not exist then you couldn't continue to query the `cmds.sets` for it so this would error.